app/components/WidgetDynamicContent.tsx

```diff
@@ -78,6 +81,10 @@ export default class WidgetDynamicContent extends Component {
         if (requiresFetch) {
             this.fetchData();
         }
+
+        if (widget.definition.postRender && this.containerRef) {
```

Comment:
We should probably check if `this.containerRef.current` is defined, instead of `this.containerRef` (the ref object is always created in the constructor).

```suggestion
        if (widget.definition.postRender && this.containerRef.current) {
```

Also, can we have some console error if `this.containerRef.current` is not defined? This would mean there is a bug in the component :smile:

Reply:
Do we not need this logic in `componentDidMount`?